### PR TITLE
DIV-6419: Add new trigger states to Events for users

### PR DESCRIPTION
**Pull request by HarryH96**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DIV-6419

### Change description ###
- Adding the new Bailiff Service Journey States into Existing Events
- "amendPetitionForRefusalSol" Event /  for [PETSOLICITOR] role
- "allowPetitionerToAmend" Event / for caseworker-divorce-courtadmin_beta, caseworker-divorce-superuser roles

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState-bailiff-nonprod.json [hunk 1]
```diff
@@ -166,5 +166,75 @@
     "CaseStateID": "IssuedToBailiff",
     "UserRole": "[RESPSOLICITOR]",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "3/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseStateID": "AwaitingAlternativeService",
```
Comment (shaunganley) [inline definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState-bailiff-nonprod.json:173]: what's the purpose of these changes if you're not adding new states?

Comment (HarryH96) [reply definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState-bailiff-nonprod.json]: As you have pointed out, these first two StateAuths are redundant, but the rest are required as users must have at least R permissions on a new State

**Review by johnoneill46 — APPROVED** [review]

**Review by ravinderpal — APPROVED** [review]
